RecoLocalMuon/Configuration/python/RecoLocalMuon_EventContent_cff.py

```diff
@@ -28,13 +28,17 @@
         'keep *_cscSegments_*_*', 
         'keep *_rpcRecHits_*_*')
 )
+from Configuration.StandardSequences.Eras import eras
+eras.run3_GEM.toModify( RecoLocalMuonFEVT, outputCommands = RecoLocalMuonFEVT.outputCommands + ['keep *_gemRecHits_*_*'] )
+eras.run3_GEM.toModify( RecoLocalMuonRECO, outputCommands = RecoLocalMuonRECO.outputCommands + ['keep *_gemRecHits_*_*'] )
+eras.run3_GEM.toModify( RecoLocalMuonAOD, outputCommands = RecoLocalMuonAOD.outputCommands + ['keep *_gemRecHits_*_*'] )
+
+eras.phase2_muon.toModify( RecoLocalMuonFEVT, outputCommands = RecoLocalMuonFEVT.outputCommands + ['keep *_me0RecHits_*_*'] )
+eras.phase2_muon.toModify( RecoLocalMuonRECO, outputCommands = RecoLocalMuonRECO.outputCommands + ['keep *_me0RecHits_*_*'] )
+eras.phase2_muon.toModify( RecoLocalMuonAOD, outputCommands = RecoLocalMuonAOD.outputCommands + ['keep *_me0RecHits_*_*'] )
+
+eras.phase2_muon.toModify( RecoLocalMuonFEVT, outputCommands = RecoLocalMuonFEVT.outputCommands + ['keep *_me0Segments_*_*'] )
+eras.phase2_muon.toModify( RecoLocalMuonRECO, outputCommands = RecoLocalMuonRECO.outputCommands + ['keep *_me0Segments_*_*'] )
+eras.phase2_muon.toModify( RecoLocalMuonAOD, outputCommands = RecoLocalMuonAOD.outputCommands + ['keep *_me0Segments_*_*'] )
```

Comment:
The two groups of  `eras.phase2_muon.toModify` should be put into just one. E.g.
```python
eras.phase2_muon.toModify( RecoLocalMuonFEVT, outputCommands = RecoLocalMuonFEVT.outputCommands + ['keep *_me0RecHits_*_*', 'keep *_me0Segments_*_*'] )
```
or better yet
```python
_additionalMuonKeeps = ['keep *_me0RecHits_*_*', 'keep *_me0Segments_*_*']
eras.phase2_muon.toModify( RecoLocalMuonFEVT, outputCommands = RecoLocalMuonFEVT.outputCommands +_additionalMuonKeeps )
eras.phase2_muon.toModify( RecoLocalMuonRECO, outputCommands = RecoLocalMuonRECO.outputCommands +_additionalMuonKeeps )
eras.phase2_muon.toModify( RecoLocalMuonAOD, outputCommands = RecoLocalMuonAOD.outputCommands +_additionalMuonKeeps )
```
Or even better yet
```python 
def _updateOutput( era, outputPSets, commands):
   for o in outputPSets:
      era.toModify( o, outputCommands = o.outputCommands + commands )

_outputs = [RecoLocalMuonFEVT, RecoLocalMuonRECO, RecoLocalMuonAOD]
_updateOutput( eras.run3_GEM, _outputs, ['keep *_gemRecHits_*_*'] )
_updateOutput(eras.phase2_muon, _outputs, ['keep *_me0RecHits_*_*', 'keep *_me0Segments_*_*'])
```
This last one guarantees that you can't accidently start from the `outputCommands` from a different `PSet`